model/index.js

```diff
@@ -64,13 +60,36 @@ module.exports = yeoman.Base.extend({
 
   addNullDataSourceItem: actions.addNullDataSourceItem,
 
+  setBluemixDatasourceState: function() {
+    if (this.options.bluemix) {
+      var configPath = this.destinationPath('.bluemix/datasources-config.json');
+      if (!fs.existsSync(configPath)) {
+        this.log('datasources-config.json not found');
+        process.exit();
```

Comment:
Yes, I agree with the points.

No prompts are presented when `process.exit()` is called at that point, so it should be a clear exit. What other ways could this mess up the terminal?

Showing `done(err)`'s stack trace for that particular condition doesn't look like good UI. However, I decided to use `done(err)` on line no 85 to make the code testable.